### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by NotBhalu** [audiamus/AaxAudioConverter#152] [General]
I loved your work and really appreciate it, so i just wanted to know if there is any way to donate

**Comment by audiamus** [comment]
Thank you very much for your question. Please don't feel offended but I want AAX Audio Converter and its companion Book Lib Connect to be and remain completely free software, with no hidden or moral obligations. You can best help AAX Audio Converter and Book Lib Connect by spreading your user experience.
